src/PKSim.Core/Services/PKAnalysesTask.cs

```diff
@@ -193,7 +193,17 @@ public string DescriptionFor(string pkParameterName)
          return pkParameter.Description;
       }
 
-      private IndividualPKAnalysis calculatePKFor(DataColumn dataColumn, string moleculeName, PKCalculationOptions options, GlobalPKAnalysis globalPKAnalysis = null)
+      private IndividualPKAnalysis calculatePKForPopulation(DataColumn dataColumn, string moleculeName, PKCalculationOptions options, GlobalPKAnalysis globalPKAnalysis = null)
```

Comment:
Why do we have two methods with the same signature, the same parameters, and calling the same method returning the same value? I don't undertand